plugins/csv/unit_test/test_json_to_csv_bytes.py

```diff
@@ -0,0 +1,93 @@
+        }
+        self.assertEqual(actual, expected)
+
+    def test_json_to_csv_bytes_empty_json(self):
+        action = JsonToCsvBytes()
+        actual = action.run(
+            {
+                Input.JSON: [],
+            }
+        )
+        expected = {Output.CSV_BYTES: ""}
+        self.assertEqual(actual, expected)
+
+    def test_json_to_csv_bytes_value_as_array(self):
+        action = JsonToCsvBytes()
+        actual = action.run(
+            {
+                Input.JSON: [
+                    {"column1": "value1", "column2": "value2", "column3": "value3"},
+                    {"column1": "value4", "column2": ["value", "value"], "column3": "value6"},
+                ],
+            }
+        )
+        expected = {
+            Output.CSV_BYTES: "Y29sdW1uMSxjb2x1bW4yLGNvbHVtbjMNCnZhbHVlMSx2YWx1ZTIsdmFsdWUzDQp2YWx1ZTQsIlsn\ndmFsdWUnLCAndmFsdWUnXSIsdmFsdWU2DQo=\n"
+        }
+        self.assertEqual(actual, expected)
+
+    def test_json_to_csv_bytes_value_as_object(self):
+        action = JsonToCsvBytes()
+        actual = action.run(
+            {
+                Input.JSON: [
+                    {"column1": "value1", "column2": "value2", "column3": "value3"},
+                    {"column1": "value4", "column2": {"column2_1": "value", "column": "value"}, "column3": "value6"},
+                ],
+            }
+        )
+        expected = {
+            Output.CSV_BYTES: "Y29sdW1uMSxjb2x1bW4yLGNvbHVtbjMNCnZhbHVlMSx2YWx1ZTIsdmFsdWUzDQp2YWx1ZTQsInsn\nY29sdW1uMl8xJzogJ3ZhbHVlJywgJ2NvbHVtbic6ICd2YWx1ZSd9Iix2YWx1ZTYNCg==\n"
+        }
+        self.assertEqual(actual, expected)
+
+    def test_json_to_csv_bytes_empty_object(self):
+        action = JsonToCsvBytes()
+        actual = action.run(
+            {
+                Input.JSON: [
+                    {"column1": "value1", "column2": "value2", "column3": "value3"},
+                    {"column1": "value4", "column2": "value5", "column3": "value6"},
+                    {},
+                ],
+            }
+        )
+        expected = {
+            Output.CSV_BYTES: "Y29sdW1uMSxjb2x1bW4yLGNvbHVtbjMNCnZhbHVlMSx2YWx1ZTIsdmFsdWUzDQp2YWx1ZTQsdmFs\ndWU1LHZhbHVlNg0KDQo=\n"
+        }
+        self.assertEqual(actual, expected)
+
+    def test_json_to_csv_bytes_empty_fields(self):
+        action = JsonToCsvBytes()
+        actual = action.run(
+            {
+                Input.JSON: [
+                    {"column1": "value1", "column2": "", "column3": "value3"},
+                    {"column1": "", "column2": "value5", "column3": "value6"},
+                    {"column1": "value7", "column2": "value8", "column3": ""},
+                ],
+            }
+        )
+        expected = {
+            Output.CSV_BYTES: "Y29sdW1uMSxjb2x1bW4yLGNvbHVtbjMNCnZhbHVlMSwsdmFsdWUzDQosdmFsdWU1LHZhbHVlNg0K\ndmFsdWU3LHZhbHVlOCwNCg==\n"
+        }
+        self.assertEqual(actual, expected)
```

Comment:
Put a unit test in here for unstructured data. So like 

[col1, col2, col3]
[col1, col2]
[col1, col2, col3, col4]

Reply:
Added!